dashboard/ui.R

```diff
@@ -0,0 +1,72 @@
+shinyUI(navbarPage(
+  title = "Data Quality Dashboard",
+  theme = "styles.css",
+  tabPanel( #at the top of every page to navigate through the entire dashboard, contains tabs for home
+    title = "Home",
+    navlistPanel(
+       id = "tabset",
```

Comment:
`id` should be unique - this carries across all instances of this throughout the ui.